candid/src/parser/value.rs

```diff
@@ -319,6 +319,60 @@ impl IDLValue {
     }
 }
 
+pub mod pretty {
```

Comment:
@chenyan-dfinity Is it okay to leave this here (this file/module), and in this PR?  WDYT?

Reply:
Yeah, it's fine to do in this PR. Eventually, we can replace the `Display` trait with this pretty version.